theano/tensor/tests/test_opt.py

```diff
@@ -3239,6 +3242,16 @@ def test_local_subtensor_of_alloc():
                 assert xval.__getitem__(slices).shape == val.shape
 
 
+def test_local_inplace_incsubtensor1():
+    # Test local_inplace_incsubtensor1 for stack trace
+    x = fmatrix()
+    y = x**2
+    a = lmatrix()
+    z = advanced_inc_subtensor1(y, a, [1,3])
+    f = theano.function([x, a], z)
+    assert check_stack_trace(f, ops_to_check='all')
+
+
```

Comment:
You should move that to `test_subtensor.py`.
Make sure that the adv_inc_subtensor1 in the graph is actually inplace.